### PR TITLE
Auth test

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Added dummy credentials to login
Added form validation in signup page

### CODE REVIEW

File: src/pages/Auth/Auth.css
Comment (Kothalkarpriya) [inline src/pages/Auth/Auth.css:56]: less use of !important will be good for future complications with styling

File: src/pages/Auth/Auth.css
Comment (Kothalkarpriya) [inline src/pages/Auth/Auth.css:125]: you used !important a lot, please use stylings using the order it might reduce the use of !important

File: src/pages/Auth/Login.js
Comment (Kothalkarpriya) [inline src/pages/Auth/Login.js:28]: Remove unwanted commented code

File: src/pages/Auth/SignUp.js
Comment (Kothalkarpriya) [inline src/pages/Auth/SignUp.js:19]: here also you should remove unwanted commented code

**Review by Kothalkarpriya — APPROVED** [review]
Look into my suggestions once, Your code is clean and good to merge after the changes suggested

File: src/pages/Auth/Login.js
Comment (Aditi0505) [inline src/pages/Auth/Login.js:16]: remove commented code

File: src/App.js
Comment (Aditi0505) [inline src/App.js:10]: remove commented code

File: src/pages/Auth/Login.js
Comment (Aditi0505) [inline src/pages/Auth/Login.js:32]: remove commented code

File: src/pages/Auth/Login.js
Comment (Aditi0505) [inline src/pages/Auth/Login.js:49]: remove this if not needed.

File: src/pages/Auth/SignUp.js
Comment (Aditi0505) [inline src/pages/Auth/SignUp.js:33]: remove commented code

File: src/pages/index.js
Comment (Aditi0505) [inline src/pages/index.js:5]: remove commented code

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions for you to look at. 
- Instead of using !important, you can increase specificity and use it. It will override the existing styling of CL.

File: src/pages/Auth/Auth.css
Comment (ranitasahaa1230) [reply src/pages/Auth/Auth.css:125]: Sure will modify it

**Comment by ranitasahaa1230** [comment]
> Left few suggestions for you to look at.
> 
> * Instead of using !important, you can increase specificity and use it. It will override the existing styling of CL.

**Comment by ranitasahaa1230** [comment]
Sure will do it

**Pull request merged**
